tests/Utils/Test_IPConverter.cpp

```diff
@@ -45,8 +45,8 @@ BOOST_AUTO_TEST_CASE(test_IPNumericaltoString) {
 BOOST_AUTO_TEST_CASE(test_IPStringToNumerical) {
   INIT_STDOUT_LOGGER();
 
-  boost::multiprecision::uint128_t result =
-      IPConverter::ToNumericalIPFromStr("127.0.0.1");
+  boost::multiprecision::uint128_t result;
+  IPConverter::ToNumericalIPFromStr("127.0.0.1", result);
```

Comment:
Should check for the result of `ToNumericalIPFromStr` wherever it's called

Reply:
Done